python-test-samples/apigw-lambda-dynamodb/tests/unit/mock_test.py

```diff
@@ -41,7 +41,7 @@ def setUp(self) -> None:
         # This simple technique works if there are no intrinsics (like !If or !Ref) in the
         # resource properties for KeySchema, AttributeDefinitions, & BillingMode.
         sam_template_table_properties = self.read_sam_template()["Resources"]["DynamoDBTable"]["Properties"]
-        self.mock_dynamodb = boto3.resource("dynamodb", region_name="us-east-1")
+        self.mock_dynamodb = boto3.resource("dynamodb")
```

Comment:
The region is not required for mocking, but specifying it makes the tests fail when run in another region.